src/EnergyPlus/MixedAir.cc

```diff
@@ -4286,6 +4287,7 @@ namespace MixedAir {
 		// save the min outside air flow fraction and max outside air mass flow rate
 		if ( AirLoopNum > 0 ) {
 			AirLoopFlow( AirLoopNum ).OAMinFrac = OutAirMinFrac;
+			AirLoopFlow( AirLoopNum ).MinOutAir = OutAirMinFrac * OAController( OAControllerNum ).MixMassFlow;
```

Comment:
I can't see anything below here that would be affected by this change, in fact, it looks like it would correct things when MixMassFlow < OutAirMinFrac * AirLoopFlow( AirLoopNum ).DesSupply. Not sure what line 4052 is used for now.